spm_results_export.m

```diff
@@ -28,14 +27,12 @@ function spm_results_export(SPM,xSPM,TabDat)
     if exist(fullfile(SPM.filepath,'html')) ~= 7 

```

Comment:
Hi @TomMaullin: have you looked at this comment, yet? Apart from this I think the PR is ready to merge!